site/content/en/docs/concepts/_index.md

```diff
@@ -89,4 +89,9 @@ resources that are now required by the owning ClusterQueue.
 
 ### [Fair Sharing](/docs/concepts/fair_sharing)
 
-Mechanisms in Kueue to share quota between tenants fairly.
\ No newline at end of file
+Mechanisms in Kueue to share quota between tenants fairly.
+
+### [Elastic Workloads](/docs/concepts/elastic_workload)
```

Comment:
It would probably be good to somehow correlate the docs for `partial admission and dynamic reclaim`. And differentiate these features so people don't get confused which stage of scheduling dynamic scaling happens.

Reply:
Excellent callout @amy. 

I added a "partial admission" limitation to the section below. 

As for the 'dynamic reclaim` - I may need a little guidance. Elastic Jobs are not relying on or intersecting with the `dynamic reclaim` feature/functionality. What would you recommend?